code/game/machinery/doors/door.dm

```diff
@@ -283,6 +283,7 @@
 	set_opacity(0)
 	sleep(5)
 	density = FALSE
+	flags_1 ^= PREVENT_CLICK_UNDER_1

```

Comment:
`^` isn't the operator you want, this will add `PREVENT_CLICK_UNDER_1` if it didn't have it before
```suggestion
	flags_1 &= ~PREVENT_CLICK_UNDER_1
```

Reply:
Done